src/main/python/utils/dynamo_s3_csv_exporter.py

```diff
@@ -0,0 +1,76 @@
+## Job to extract information from DynamoDB, write it into a CSV file and add it to S3.
+import csv
+import json
+import decimal
+import boto3
+import os
+from botocore.exceptions import NoCredentialsError
+
+class DecimalEncoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, decimal.Decimal):
+            if o % 1 > 0:
+                return float(o)
+            return int(o)
+        return super(DecimalEncoder, self).default(o)
+
+def extract_and_transform():
+    dynamo_db = boto3.resource('dynamodb')
+    table = dynamodb.Table('survey-app-dump')
+
+    response = table.scan()
+
+    raw_data = response['Items']
+    
+    while 'LastEvaluatedKey' in response:
+        response = table.scan(ExclusiveStartKey=response['LastEvaluatedKey'])
+        raw_data.extend(response['Items'])
+    
+    json_data = map(lambda item: json.dumps(item, cls=DecimalEncoder), raw_data)
+
+    return json_data
```

Comment:
I think this will still pull all the data into memory. What we should do is, scan until 128MB, create a file then scan the next block. if LastEvaluatedKey scanned is <128 and more items in table, scan next block. So, some checks needed here. There can be scenarios where adding a next block will make files bigger, in that case we can skip and create file and add the the block for next file. something like that.?